src/solc_0.7/catalyst/GemsCatalystsRegistry.sol

```diff
@@ -75,21 +75,25 @@ contract GemsCatalystsRegistry is WithSuperOperators {
         uint256[] calldata amounts
     ) public {
         for (uint256 i = 0; i < gemIds.length; i++) {
-            burnGem(from, gemIds[i], amounts[i]);
+            if (amounts[i] != 0) {
```

Comment:
The conditional here prevents `_burn` being called when `amount == 0` (which causes execution to fail at:
`require(amount > 0, "BURN_O_TOKENS");` )

Reply:
I think that added condition makes sense, as there are a lot of places in code we decided not to revert for zero values, so it's pretty consistent.